src/lib.rs

```diff
@@ -124,20 +164,34 @@ impl Pool {
         // Submit the new thread to the thread waitgroup.
         pool.wait.submit();
 
+        let thread_number = self.inner.thread_counter.fetch_add(1, Ordering::SeqCst);
+
+        // Deal with thread configuration.
+        let mut builder = thread::Builder::new();
+        if self.inner.thread_config.name_prefix.is_some() {
```

Comment:
how about `if let Some(prefix) = self.inner.thread_config.name_prefix.as_ref())`, same for stack_size